public/app/features/manage-dashboards/components/DeletePublicDashboard/DeletePublicDashboardModal.tsx

```diff
@@ -0,0 +1,49 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data/src';
+import { ConfirmModal, useStyles2 } from '@grafana/ui/src';
+
+const Body = ({ title }: { title?: string }) => {
+  const styles = useStyles2(getStyles);
+
+  return (
+    <>
+      <p className={styles.title}>Do you want to delete this public dashboard?</p>
+      <p className={styles.description}>
+        {title
+          ? `Public dashboard configuration will be deleted, while "${title}" dashboard will remain private.`
+          : 'Public dashboard configuration will be deleted'}
```

Comment:
what do you think about this text?

Reply:
I'm not sure but I don't have better ideas. Maybe @owensmallwood or @jalevin could help us here.

Reply:
Are we still calling it a `configuration`? I think it's important to be explicit about whats going on. I would have went with something like `This will delete your public dashboard. The original dashboard will not be deleted.`

Reply:
I have a PR coming soon that removes "config" and "configuration" in comments, method names, etc

Reply:
so, do we agree to change the text to 
`This will delete your public dashboard. The original dashboard will not be deleted.` ?
What about `Public dashboard configuration will be deleted. Original "${title}" dashboard will not be deleted`